powerline/segments/common/players.py

```diff
@@ -477,3 +477,54 @@ def get_player_status(self, pl):
 
 {0}
 ''').format(_common_args.format('rdio')))
+
+
+class ITunesPlayerSegment(PlayerSegment):
+	def get_player_status(self, pl):
+		status_delimiter = '-~`/='
+		ascript = '''
+			tell application "System Events"
+				set process_list to (name of every process)
+			end tell
+		
+			if process_list contains "iTunes" then
+				tell application "iTunes"
+					if player state is playing then
+						set t_title to name of current track
```

Comment:
I am not sure.... Again I believe this exist in the previous Powerline segments or somewhere online I can't recall. I took most part of it and modified some parts to make it work.